docs/source/datasets.md

```diff
@@ -120,8 +120,8 @@ run_info = submit_to_azure_if_needed(...,
 input_folder = run_info.input_datasets[0]
 ```
 
-Secondly, you can check the returned path in `run_info`, and replace it with something for local execution. 
-`run_info.input_datasets[0]` will be `None` if the script runs outside of AzureML, and no `local_folder` is available.
+Secondly, if `local_folder` is not specified, then if an AzureML workspace cannot be found then `run_info.input_datasets[0]` will be `None` and you can check the returned path in `run_info`, and replace it with something for local execution. 
```

Comment:
How about we just kill this part here? Or update the whole text to clearly say "there are 3 paths to success" (and maybe structure them with subheadings

Reply:
I think we should kill the middle part. First option could be to put local path into DAtasetConfig constructor, second is automagic download/mount.

Reply:
Ok, I've simplified that